README.md

```diff
@@ -1,14 +1,23 @@
 # Simple Bank Exercise
 
-The Simple Bank directory is a truffle project that contains the required contract, migration and test files. In this exercise you are going to implement the SimpleBank.sol contract.
+SimpleBank is a [Truffle](https://www.trufflesuite.com/) project that contains the required contract, [migration](https://www.trufflesuite.com/docs/truffle/getting-started/running-migrations#migration-files) and [Truffle JavaScript test files](https://www.trufflesuite.com/docs/truffle/testing/writing-tests-in-javascript). In this exercise you are going to implement the SimpleBank.sol contract.
 
-The contract contains the framework and comments to implement the contract. Follow the comments outlined in the SimpleBank.sol contract to implement the contract. We have written a set of tests to determine if you implement the contract correctly. 
+The contract contains the framework and comments to implement the contract. Follow the comments outlined in the SimpleBank.sol contract to implement its functionality. We have written a set of tests to determine correct contract behavior as well as guide you through the implementation. 
 
-To test your implementation start your development blockchain by running truffle develop. From the truffle console, compile and migrate the contracts by running compile and migrate. You can run the pre-written tests by running test.
 
-If your tests do not pass, modify the contract, recompile, redeploy and retest. Iterate until all of the tests pass.
+TODO: update this flow
+The general workflow is:
+  - run tests by issuing the `truffle test` from the command line
+  - see them fail, 
+  - then update the contract, 
+  - see them pass
 
+  
+Proceed by opening the **SimpleBank** contract in your editor, along with a **terminal console** from where you can run truffle tests. Run the tests, (`truffle test`), examine the output then modify the contract to make the tests pass. If tests do not pass, modify the contract, retest and iterate until all are passing.
 
-### A note on testing
 
-Check out the testing files to see how tests are implemented in JavaScript. We will go over the details of implementing tests later in the course.
+### Note on testing
+
+ ** Check out the testing files to see how tests are implemented in JavaScript.
+    We will go over the details of implementing tests later in the course. 
+ ** If you feel there is a lot of noise, 
```

Comment:
Appreciate this, you're waiting for the go-ahead to add about how to skip tests?

Reply:
oops, missed that